modules/processing.py

```diff
@@ -615,7 +615,7 @@ def sample(self, conditioning, unconditional_conditioning, seeds, subseeds, subs
 class StableDiffusionProcessingImg2Img(StableDiffusionProcessing):
     sampler = None
 
-    def __init__(self, init_images=None, resize_mode=0, denoising_strength=0.75, mask=None, mask_blur=4, inpainting_fill=0, inpaint_full_res=True, inpaint_full_res_padding=0, inpainting_mask_invert=0, **kwargs):
+    def __init__(self, init_images: list=None, resize_mode: int=0, denoising_strength: float=0.75, mask: str=None, mask_blur: int=4, inpainting_fill: int=0, inpaint_full_res: bool=True, inpaint_full_res_padding: int=0, inpainting_mask_invert: int=0, **kwargs):

```

Comment:
Can you gut check these types? The one I wasn't really sure about was `mask`

Reply:
mask is a PIL image

Reply:
I don't have any exposure to pydantic, but it seems like it was struggling with the `Optional[Image]` and the `Image` types. I went with `Any`, but another option is to create an [annotated type](https://pydantic-docs.helpmanual.io/usage/types/#annotated-types) around the Image class

Reply:
I think the Any type is fine in this case